### ISSUE TITLE
lightning_strike_time topic name

### ISSUE THREAD

**Issue by simonranson** [briis/hass-weatherflow2mqtt#13]
I was just taking a look at the .storage.json file to add some real-world values for last rain start and last lightning to clear the 1970 date from HA. I noticed there that one is stored as ISO time/date and the other is epoch - not sure if there a reason for this?

The lightning_strike_time topic has the name "Last Lightning Today". I think "Today" can be removed from this. The sensor table doesn't say this is reset every day at midnight.

Cheers
Simon

**Comment by briis** [comment]
Hi Simon
> I noticed there that one is stored as ISO time/date and the other is epoch - not sure if there a reason for this

Honestly, this is by chance. I was working on one thing first where I used one format, and then the other format a few days later. When I found out, I didn't bother to change it.

The overall issue with these storage files, is that what we get from the station are just the raw data, so if I wanted to simulate the APP somehow, I needed a place to store some values between updates, at the same time, I did not really want to create a whole database system underneath, as Home Assistant would take care of the historic data. So the whole approach is not perfect, but it works somehow.
And when you first run the system, I need some initial values - and when it comes to math with dates, most systems are very sensitive, so just storing nothing, gave issues, thats why I used the 1970 date.

> The lightning_strike_time topic has the name "Last Lightning Today". I think "Today" can be removed from this. The sensor table doesn't say this is reset every day at midnight.

I already changed the name in the latest release to `Last Lightning Strike`
